cmd/csi-snapshotter/main.go

```diff
@@ -125,6 +125,9 @@ func main() {
 		klog.Fatal("Error while parsing feature gates: ", err)
 	}
 
+	// Enable VGS unconditionally
+	utilfeature.DefaultMutableFeatureGate.OverrideDefault(features.VolumeGroupSnapshot, true)
```

Comment:
Lets enable it at the deployment time not to have this changed in the repo itself?

Reply:
You mean by providing this featuregate by default in args?

Reply:
yes when deploying we will set the existing VGS flag and enable the feature so that we will have more control over it.

Reply:
@Nikhil-Ladha FYI ^^

_We'll provide the feature flag by default in deployment manifest_

Reply:
Yeah, the ceph-csi-op PR takes care of doing that.